app/models/miq_schedule_worker/runner.rb

```diff
@@ -400,7 +400,7 @@ def rufus_add_normal_schedule(options)
   def rufus_add_monthly_schedule(options)
     months = options.delete(:months)
     method = options.delete(:method)
-    interval = options.delete(:interval)
+    options.delete(:interval)
```

Comment:
This likely does nothing (options used below likely don't depend on this deletion) and can be removed.